### PR TITLE
Project 10 Maidelin Rubio

### PR DESCRIPTION
**Pull request by rubmaigut**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Review by FD000 — COMMENTED** [review]
Fantastic work, I was very impressed by your code. One of the most beautiful project I have ever seen. 

I would like you to review 1 thing in your project:

1) When you are answering questions, there is a counter in the bottom that informs you what question you are at, that one counts wrong. For instance when you have answered first question, it says "3 answered of 5".

**Comment by rubmaigut** [comment]
> Fantastic work, I was very impressed by your code. One of the most beautiful project I have ever seen.
> 
> I would like you to review 1 thing in your project:
> 
> 1. When you are answering questions, there is a counter in the bottom that informs you what question you are at, that one counts wrong. For instance when you have answered the first question, it says "3 answered of 5".

Hi @ FD000, when you answered the first question as "yes", the code avoids the protein question because all the answers are "non-vegan" options. So if your answer is "yes", the remaining questions are 3 and you just choose 3 more options to see the results. when your answer is "No", you need to choose 4 options more but always the length of the array is 5 questions.

File: code/src/Data.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        "lamb",
+        "meat",
+        "Pork"
+      ]
+    },
+    {
+      "id": 2,
+      "question": "Exclude one of the following... ",
+      "filter": "health",
+      "optionType": "radio",
```
Comment (KarolCislowski) [inline code/src/Data.json:31]: I think the checkbox would be a better option here. 🤓

**Review by KarolCislowski — APPROVED** [review]
It is an awesome project! 
And now I need to go to eat something because it made me hungry 🥘

**Pull request closed without merging**